pico_project/pico_project.py

```diff
@@ -1101,6 +1101,9 @@ def generateProjectFiles(projectPath, projectName, sdkPath, projects, debugger):
 
 
 def LoadConfigurations():
+    # open the pico_configs.tsv from the executable's directory
+    this_module_location = os.path.dirname(os.path.realpath(__file__))
+    tsv_file = os.path.join(this_module_location, "pico_configs.tsv")
```

Comment:
This bit probably needs updating in light of the new `--tsv` option. Note how the line below does `open(args.tsv)` and not `open(tsv_file)`. I suspect the correct fix might be to move the `this_module_location ...` stuff into `ParseCommandLine` ?

Reply:
Done as suggested. Tested works OK.